.github/workflows/build.yml

```diff
@@ -9,6 +9,10 @@ jobs:
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```

Comment:
Why do we use two different techniques for caching? Can we unify them?

Reply:
I was following the platform. One cache is typical for node modules, second is for `setup-node@v2` to be honest I did not analyze the difference, I assumed that this is deliberate in the platform.

Reply:
IMO, it's a mistake in `wnl-platform` that we shouldn't follow

Reply:
Ok.

Reply:
Now, we don't use any cache in `ci.yaml`. What I was forcing is: cache unification across different workflows in this repo, not removal of the cache

Reply:
Also, I suggest using https://github.com/actions/setup-node#caching-global-packages-data - it's dedicated for Node, so it's possible github optimized it for this use-case